sessions/recap-project-6/example-implementation-with-comments/components/Comments.js

```diff
@@ -0,0 +1,101 @@
+import styled from "styled-components";
+import { useRouter } from "next/router.js";
+import { FormContainer, Input, Label } from "./Form.js";
+import { StyledButton } from "./StyledButton.js";
+import useSWR from "swr";
+import { Fragment } from "react";
+
+const Article = styled.article`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  border: 1px solid black;
+  border-radius: 0.8rem;
+  padding: 0.5rem;
+  text-align: center;
+  p {
+    border-bottom: solid 1px black;
+    padding: 20px;
+  }
```

Comment:
We teach our students to avoid nested css rulesets, since it changes the specificity of the rule. I think this should be its own styled component.